test/common/tmpdir.js

```diff
@@ -14,6 +14,7 @@ const testRoot = process.env.NODE_TEST_DIR ?
 // Using a `.` prefixed name, which is the convention for "hidden" on POSIX,
 // gets tools to ignore it by default or by simple rules, especially eslint.
 const tmpdirName = '.tmp.' +
+  (process.env.BUILD_ID || process.pid + '.') +
```

Comment:
Going to check this gets passed - my understanding in test.py is that the os environment variables are copied and accessible.